### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by IzzySoft** [amirzaidi/DNGProcessor#2]
Looks like your app is perfectly suited to be on [F-Droid](https://f-droid.org/) – an Android "market" for free and open source apps. I've openened [an RFP](https://gitlab.com/fdroid/rfp/issues/892) (request for packaging) there, and hope you support the inclusion there? F-Droid would then build the app from the source you provide here – and automatically check for updates (by the tags you create; I've assumed you named then `v<versionCode>`?)

**Comment by amirzaidi** [comment]
Yeah sure, if it doesn't require any additional work from my end I'm fine with that. I label every release I make v(n) where n is the release, the only one so far is v1.

**Comment by IzzySoft** [comment]
Thanks Amir! And I don't think it will require extra work from your side. If you keep `n = versionCode`, everything should run fine with future updates.

**Comment by IzzySoft** [comment]
PS: One thing you might wish to check for your own builds is [an issue with your gradlew](https://gitlab.com/fdroid/rfp/issues/892#note_139408779). You certainly want to secure that 😉

**Comment by amirzaidi** [comment]
@IzzySoft should be resolved by https://github.com/amirzaidi/DNGProcessor/commit/23495aed648b0bb41079999b6f22bb0b43c3c8f1

**Comment by IzzySoft** [comment]
Thanks, @amirzaidi! Above mentioned RFP currently is waiting for one of our "integrators" to pick it up (I've marked it "ready" about 2 weeks agi). We're chronically understaffed at F-Droid, so apologies that it might take a little…

**Comment by Benjamin-Loison** [comment]
For reference the app is now on F-Droid: https://f-droid.org/en/packages/amirz.dngprocessor/
